q2_feature_table/_sample.py

```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import biom
+
+
+def sample(table: biom.Table, sampling_depth: int, axis: str) -> biom.Table:
```

Comment:
The current API naming could be confusing because the feature table has samples, so this verbiage suggests that users *sample the samples* (if `axis='sample'`). The parameter `sampling_depth` is also used in a different context within qiime2 (i.e. for rarefaction depth).

@gregcaporaso and I chatted about this API, what do you think?

```python
subsample(table, n, axis):
```

Reply:
:+1: agree it was a bit confusing even if accurate :) Will revise in a moment